.vscode/launch.json

```diff
@@ -0,0 +1,47 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+		{
+			"name": "Attach to kind-control-plane-1",
+			"type": "go",
+			"request": "attach",
+			"mode": "remote",
+			"remotePath": "",
```

Comment:
These and the similar are redundant and can be removed. Setting subsitutePath is the alternative, which ware doing.

Reply:
These meaning just `remotePath`? I can try dropping that. I'm guessing type/request/mode are all still required?

Reply:
Yup "these" are "remotePath" values. Shouldnt be necessary